java/ql/src/semmle/code/java/security/XPath.qll

```diff
@@ -0,0 +1,64 @@
+/** Provides classes to reason about XPath vulnerabilities. */
+
+import java
+import semmle.code.java.dataflow.FlowSources
```

Comment:
Possibly move this import to the query. It doesn't look like it's used here.

Reply:
You are right, fixed in ea74205.